src/main/java/com/esri/samples/geometry/convex_hull_list/ConvexHullListSample.java

```diff
@@ -0,0 +1,255 @@
+      pointCollection2.add(new Point(6986529.4575743, 354646.16535905));
+      pointCollection2.add(new Point(5319692.48038653, 1205796.96222089));
+
+      // create a polygon geometry from the point collection
+      Polygon polygon2 = new Polygon(pointCollection2);
+      // create the graphic for polygon2
+      Graphic polygonGraphic2 = new Graphic(polygon2, polygonFill);
+      // add the polygon2 graphic to the graphics overlay collection
+      graphicsOverlay.getGraphics().add(polygonGraphic2);
+
+      // create a button to create and show the convex hull
+      Button convexHullButton = new Button("Convex Hull");
+      convexHullButton.setMaxWidth(Double.MAX_VALUE);
+
+      // create a button to clear the convex hull results
+      Button resetButton = new Button("Reset");
+      resetButton.setMaxWidth(Double.MAX_VALUE);
+
+      // create a check box for toggling union option on or off
+      CheckBox checkBox = new CheckBox("Union");
+
+      convexHullButton.setOnAction(e -> {
+        // reset the convex hull graphics overlay
+        convexHullGraphicsOverlay.getGraphics().clear();
+
+        Boolean unionBool = checkBox.isSelected();
+
+        // add the geometries of the two polygon graphics to a list of geometries
+        List<Geometry> allPolygonGeometries = new ArrayList<>();
+        allPolygonGeometries.add(polygonGraphic1.getGeometry());
+        allPolygonGeometries.add(polygonGraphic2.getGeometry());
+
+        // retrieve the returned result from the convex hull operation.
+        // when unionBool = true there will be one returned convex hull geometry
+        // when = false, one convex hull geometry per input geometry.
+        List<Geometry> convexHullGeometries = GeometryEngine.convexHull(allPolygonGeometries, unionBool);
+
+        // loop through the returned geometries.
+        for (Geometry geometry : convexHullGeometries) {
+
+          // create a simple line symbol for the outline of the convex hull graphic(s)
+          SimpleLineSymbol convexHullLine = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 5);
+          // Create the simple fill symbol for the convex hull graphic(s)
+          SimpleFillSymbol convexHullFill = new SimpleFillSymbol(SimpleFillSymbol.Style.NULL, 0x00000000, convexHullLine);
+
+          // create the graphic for the convex hull(s)
+          Graphic convexHullGraphic = new Graphic(geometry, convexHullFill);
+
+          // add the convex hull(s) graphic to the convex hull graphics overlay
+          convexHullGraphicsOverlay.getGraphics().add(convexHullGraphic);
+        }
+
+        // enable reset button after convex hull button has been pressed.
+        resetButton.setDisable(false);
+        // Reset check box after convex hull button has been pressed.
+        checkBox.setSelected(false);
+      });
+
+      // disable clear button from starting application (when nothing to clear)
+      resetButton.setDisable(true);
```

Comment:
would be good to call this `clearButton` with text "Clear" like in the other convex hull sample